src/content/zh-CN/2020/performance.md

```diff
@@ -0,0 +1,396 @@
+
+从这个角度来说，CLS 与本章提到的其他指标相比，是一种不同的、新型的用户体验整体性指标。
+
+### 按设备类型查看 CLS
+
+{{ figure_markup(
+  image="performance-cls-by-device.png",
+  caption="按设备类型分类的 CLS 总性能。", 
+  description="条形图显示，超过一半的网站具有良好的 CLS，其中桌面端为54％，移动端为60％。在这两种情况下，只有21％的网站的 CLS 被评为差。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=1672696367&format=interactive",
+  sheets_gid="1270303192",
+  sql_file="web_vitals_by_device.sql"
+  )
+}}
+
+根据 Chrome 用户体验数据，桌面端和移动端设备，超过一半的网站有着良好的 CLS 评分。桌面端和移动端表现良好比例的网站略有差别（6个百分点），后者稍高一些。 我们可以推测，手机之所以在 CLS 评分中良好比例领先，是因为移动端的优化通常倾向于使站点具备更少的功能和更小的可见尺寸。
+
+### 按地理位置查看 CLS
+
+{{ figure_markup(
+  image="performance-cls-by-geo.png",
+  caption="按国家分类的 CLS 总性能",
+  description="条形图显示，排名前28位的国家/地区至少有50％的网站报告了良好的 CLS 评分。中等（需要改进）的浏览体验总体稳定在11-15％。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=1861585123&format=interactive",
+  width="645",
+  height="792",
+  sheets_gid="47865955",
+  sql_file="web_vitals_by_country.sql"
+  )
+}}
+
+CLS 在不同地域的表现总体上是好的，至少有56%的网站获得了良好的评价。对于可感知的视觉稳定性来说，这是一个极好的消息。我们可以观察到类似的国家处于领先地位，正如我们在 LCP 地理分布中看到的那样--韩国、日本、捷克、德国、波兰。相对于其他指标，比如 LCP，视觉稳定性受地理和延迟的影响较小。排名最好的国家和排名最差的国家之间的百分比差距，在 LCP 指标上达到了61%，在 CLS 指标上达到了13%。中等评分的网站比例在整个网站中相对较低，让位于整个网站中19%-29%的差体验。导致CLS差的因素有很多--在【优化累积布局转移指南】(https://web.dev/optimize-cls/)中了解如何解决这些问题。整体而言，中等评分网站的比例最低，甚至低于19%-29的最差用户体验的比例。有许多因素都可能导致较差的CLS，请在[优化累积布局偏移指南]（https://web.dev/optimize-cls/）中了解如何解决这些问题。
+
+### 按连接类型查看 CLS
+
+{{ figure_markup(
+  image="performance-cls-by-connection-type.png",
+  caption="按连接类型分类的 CLS 总性能",
+  description="条形图显示良好、需要改进和较差的 CLS 评分分布均匀。离线和4G领先，分别拥有70％和64％的良好体验",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=151288279&format=interactive",
+  sheets_gid="306222260",
+  sql_file="web_vitals_by_ect.sql"
+  )
+}}
+
+除离线和4G外，大多数连接类型的 CLS 评分都是合理的均匀分布。在离线场景下，我们可以推测网站应该是使用了 Service Workers。因此，不会因连接类型而导致下载延迟，而下载延迟却是对站点获得良好评分有着至关重要的影响。
+
+对4G得出明确的结论是具有挑战性的，但我们可以推测，也许4G+连接是作为桌面设备的一种上网方式。如果这一假设成立，网络字体和图像可能会被大量缓存，这对 CLS 测量产生积极影响。
+
+## 网页核心指标: 首次输入延迟
+
+首次输入延迟（FID）指的是用户第一次交互和浏览器能够响应该交互之间的时间。FID 是衡量网站交互性的一个很好的指标。
+
+### 按设备分类查看 FID
+
+{{ figure_markup(
+  image="performance-fid-by-device.png",
+  caption="按设备类型分类的 FID 总性能",
+  description="条形图显示了在台式机（100％）和移动设备（80％）上良好 FID 性能的百分比很高。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=2090682651&format=interactive",
+  sheets_gid="1270303192",
+  sql_file="web_vitals_by_device.sql"
+  )
+}}
+
+在如此高比例的网站都有良好的性能评分是非常少见的的。在桌面端，基于网站分布的第75个百分位中100%的网站全部获得了快速 FID 评级，这意味着经历交互延迟的人数非常少。
+
+在移动设备上，有80％的网站被评为良好。一个可能的解释是，与台式机相比，CPU 功能降低，移动设备的网络延迟（导致脚本下载和执行延迟）以及电池效率和温度限制，从而限制了移动设备的 CPU 潜力。所有这些都直接影响交互性指标，例如FID。
+
```

Comment:
```suggestion
```

Reply:
fix

Reply:
Still seeing two lines here instead of just one?

```suggestion
```